app/views/partials/details.scala.html

```diff
@@ -121,7 +121,22 @@
 
             @lifecycle.map { lifecycle =>
                 <div class="col-6">
-                    <label class="fw-bold">Status: </label>
+                    @lifecycle match {
+                        case Lifecycle(Deprecated, _, _) => {
+                            <label class="fw-bold">Status <a href="#"
+                            tabindex="0"
+                            aria-label="Status"
+                            data-bs-toggle="popover"
+                            data-bs-trigger="focus"
+                            data-bs-placement="left"
+                            data-bs-html="true"
+                            data-bs-content='Deprecated status is defined by the <a href="@repositoryDetails.githubUrl/blob/main/repository.yaml">repository.yaml</a>. To change this you will need to either remove the `deprecated` key or change its value to `false`. Changes can take up to 30 minutes to appear in the catalogue.'>
```

Comment:
Wonder if we should be vague and just say it can take a few minutes? This number will not be maintained as we change scheduler configuration etc. (It should be pretty quick on a webhook, else it can take a long time for a scheduler to catch it)